tests/playwright/tsconfig.eslint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "include": ["src/**/*.ts", "../types/**/*.d.ts"],
+  "exclude": ["node_modules"]
+}
```

Comment:
why do we need a tsconfig.eslint.json file ?

Reply:
So. The one tsconfig.json in `tests/playwright` is being used to pack/compile,etc. all the files in. But we do not want to have specs included in `dist`. Which can be done here. But then eslint complains about files being excluded. Adding specs into `eslintignore` is not right since we wanted them being checked. So the solution is to have two tsconfig files, one for packaging, the other for eslint to include all files just for check purposes.

Reply:
I think you can remove this one now